view.go

```diff
@@ -52,7 +63,20 @@ func (v *View) head(shouldAlter bool) string {
 		ddlOperation = "ALTER"
 	}
 
-	return fmt.Sprintf("%s ALGORITHM=%s DEFINER=%s %s VIEW %s AS ", ddlOperation, v.Algorithm, definer, characteristics, EscapeIdentifier(v.Name))
+	columnNames := make([]string, 0)
+
+	if v.Columns != nil && len(v.Columns) > 0 {
+		for _, col := range v.Columns {
+			columnNames = append(columnNames, EscapeIdentifier(col.Name))
+		}
+	}
+
+	columnStr := ""
+	if len(columnNames) > 0 {
+		columnStr = fmt.Sprintf(" (%s)", strings.Join(columnNames, ", "))
+	}
+
+	return fmt.Sprintf("%s ALGORITHM=%s DEFINER=%s %s VIEW %s%s AS ", ddlOperation, v.Algorithm, definer, characteristics, EscapeIdentifier(v.Name), columnStr)
```

Comment:
I understand this is just moved from existing code, but I had seen someone use a SQL escaping library in another project and am curious if that is something we should consider for tengo too. I forget what it was called.